packages/expo-updates/ios/EXUpdates/AppLauncher/EXUpdatesAppLauncherWithDatabase.m

```diff
@@ -298,7 +301,8 @@ - (void)_downloadAsset:(EXUpdatesAsset *)asset
         asset.downloadTime = [NSDate date];
         completion(nil, asset, assetLocalUrl);
       });
-    } errorBlock:^(NSError *error, NSURLResponse *response) {
+    }
+                              errorBlock:^(NSError *error) {
```

Comment:
when on a separate line xcode auto-format has this way out in space. since all the other selector args are on their own line, I prefer to have this on it's own line. And I prefer to use xcode auto-format as a source of truth (which would be awesome if we could have a lint rule for).